content/docs/1.6.0/references/settings.md

```diff
@@ -481,10 +481,10 @@ If this setting is enabled, Longhorn automatically attaches the volume and takes
 
 #### Backup Target
 
-> Examples:  
-> `s3://backupbucket@us-east-1/backupstore`  
-> `nfs://longhorn-test-nfs-svc.default:/opt/backupstore`  
-> `nfs://longhorn-test-nfs-svc.default:/opt/backupstore?nfsOptions=soft,timeo=330,retrans=3`  
+> Examples:
+> `s3://backupbucket@us-east-1/backupstore`
+> `nfs://longhorn-test-nfs-svc.default:/opt/backupstore`
+> `nfs://longhorn-test-nfs-svc.default:/opt/backupstore?nfsOptions=soft,timeo=330,retrans=3`
```

Comment:
Please don't remove the trailing spaces.  They are necessary in markdown to keep each example on its own line.

Reply:
Moved it back. That's modified by IDE and I will notice that next time.
Thanks.